### ISSUE TITLE
Can't claim proceeds - position only appears in "reporting only" mode

### ISSUE THREAD

**Issue by P-Benesch** [AugurProject/augur#10768]
**Describe the bug**

Many months ago I (0xBde4cd4eDeb6ae5824F7a9f64dDC4C5E87Ba6B6f) took a position in the market entitled "Will Donald J. Trump win the 2020 U.S. Presidential election?"  I was long "No" and the market resolves "No" over a month ago.

However, I've been unable to claim my proceeds.  For some reason, my position does not display in the interface I see when I go to augur-link.net.  My position does, however, appear when I go to augur.net and click on "start trading" but that button takes me to "reporting only" interface, and in that interface it won't let me claim proceeds.

**Screenshots**

See attached screenshots
<img width="1680" alt="Screen Shot 2021-03-04 at 10 45 54 PM" src="https://user-images.githubusercontent.com/47550171/110064613-6cfd1580-7d3b-11eb-843b-56e31f2ccf26.png">

<img width="1680" alt="Screen Shot 2021-03-04 at 10 44 33 PM" src="https://user-images.githubusercontent.com/47550171/110064568-5656be80-7d3b-11eb-8b0c-23cc741df361.png">

**Desktop (please complete the following information):**
 - OS: MacOS Big Sur, version 11.2.2 (but I've been having this program in earlier versions of MacOS)
 - Browser: Chrome
 - Version: 88.0.4324.192

**Comment by pgebheim** [comment]
Hey @P-Benesch,

So sorry about the bug. This was fixed in a more recent version of the UI than augur-link.net (which we don't control) appears to be using.

Here is a relevant thread on Discord from a community member:

https://discord.com/channels/378030344374583298/466175938086961163/815090156838715402

**Comment by P-Benesch** [comment]
Thanks for the suggestion! Unfortunately this didn’t really work for me - this is what I saw when I tried it - any thoughts on next steps?

**Comment by pgebheim** [comment]
Heya,

A new release went out yesterday. Mind trying to links from augur.net/ipfs again to see if this is working?

**Comment by P-Benesch** [comment]
Yeah no prob - I tried it and I can see the Claim Proceeds button, which takes me to a screen to confirm I want to claim proceedings (showing the transaction cost), but when I click that Claim Proceeds button the old button turns into a red “Failed” button with an “X” net to it.  Tried it a few times with the same result.

**Comment by pgebheim** [comment]
Got it. We've had a few others report that issue. I'll add it to the investigation schedule for this week.

________________________________

**Comment by P-Benesch** [comment]
Hello!  It's been about 6 weeks, so thought I'd check back in.  I'm still not able to claim proceeds from this market, and the "nuclear option" in the discord link still doesn't work.  Any thoughts on what I should do now?

**Comment by P-Benesch** [comment]
Hello! Just checking in again - by way of update, every time I click "Claim Proceeds" It says "Failed" in red
